apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/R4ExplanationOfBenefitResourceProvider.java

```diff
@@ -291,6 +298,9 @@ public Bundle findByPatient(
       // Add bene_id to MDC logs when _lastUpdated filter is in effect
       LoggingUtils.logBeneIdToMdc(beneficiaryId);
 
+      // Log number of resources to MDC
+      BfdMDC.put("resource_count", String.format("%d", eobs.size()));
```

Comment:
can you use the new LoggingUtil method?

Reply:
Thats code that shouldnt be there. Removed.